velox/dwio/common/DirectBufferedInput.cpp

```diff
@@ -274,6 +274,7 @@ std::vector<cache::CachePin> DirectCoalescedLoad::loadData(bool isPrefetch) {
   }
   input_->read(buffers, requests_[0].region.offset, LogType::FILE);
   ioStats_->read().increment(size);
+  ioStats_->incRawBytesRead((size > overread) ? size - overread : 0);
```

Comment:
I think we need to fix the calculation of `size` here.  `size += request.loadSize` instead of `size += std::min<int32_t>(loadQuantum_, region.length)` above probably.  Do you think that makes sense?